python/README.md

```diff
@@ -0,0 +1,27 @@
+# Python Arrow Flight Client Exmaple
+This lightweight Python client connects to the Dremio Arrow Flight server endpoint. This client requires the username and password for authentication. Developers can use admin or regular user credentials for authentication. This client can query any datasets in Dremio that are accessible by the provided Dremio user. By default, the hostname is `localhost` and the port is `32010`. Developers can change these default settings by providing the hostname and port as arguments when running the client. Moreover, the tls option can be provided to establish an encrypted connection. 
```

Comment:
Wording is awkward, as variations of "This client" are used to start 3 of the initial 4 sentences.

Reply:
Changed wording, please check.